core/types/stop_orders.go

```diff
@@ -362,9 +367,11 @@ func (s *StopOrder) String() string {
 		s.Expiry.String(),
 		s.Trigger.String(),
 		s.Status,
-		s.CreatedAt,
-		s.UpdatedAt,
+		s.CreatedAt.UTC(),
```

Comment:
Why?

Reply:
for some reason the string was getting with GMT for the test so just forcing it to UTC to match the expectation, it shouldn't matter, it's only the string().